homeassistant/components/media_player/philips_js.py

```diff
@@ -106,6 +115,7 @@ def select_source(self, source):
             self._source = source
             if not self._tv.on:
                 self._state = STATE_OFF
+            self._watching_tv = bool(self._source == 'Watch TV')
```

Comment:
Since I don't have a Philips anymore I can't verify, but I believe the string ```Watch TV``` is dependent on the language the TV is set to. So this code would probably only work for some users.
I'm not sure how this could be handled efficiently. But more versatile would be something like ```self._source not in ['HDMI 1', 'HDMI 2', 'VGA'...]```